### PR TITLE
White theme fixes (Restore different grays #6750)

### PR DESCRIPTION
**Pull request by cryham**
Fixes for issue #6750.
Points 1 and 2. Alternating revision line colors and splitters (resize grips).

## Proposed changes

- BTW I moved ColorHelper and HslColor from GitUI tp GitExtUtils/GitUI, so that ColorHelper will be visible in GitExtUtils/GitUI/ControlDpiExtensions.cs
- Splitter color is identical to old one.
- Alternating line is just a bit darker (I didn't notice it much in original). Value for original color in comment.

## Test methodology

- Manual

## Test environment(s)

- Windows 7 SP1

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by gerhardol** [comment]
I _preferred_ the standard row banding that the dark theme PR introduced. I can live with changing this. The resize grips looks better in both the dark/light I test with..

Screenshots for me.

image

image

File: GitExtUtils/GitUI/ColorHelper.cs [hunk 1]
```diff
@@ -25,5 +25,11 @@ public static bool IsLightTheme()
         {
             return IsLightColor(SystemColors.Window);
         }
+
+        public static Color SplitterColor()
```
Comment (RussKie) [inline GitExtUtils/GitUI/ColorHelper.cs]:
```suggestion
        public static Color GetSplitterColor()
```

**Comment by RussKie** [comment]
> I _preferred_ the standard row banding that the dark theme PR introduced.

Is there a screenshot to compare?

**Comment by RussKie** [comment]
LGTM, just need to fix the name of the method.

**Comment by cryham** [comment]
> 
> 
> LGTM, just need to fix the name of the method.

Done in previous commit.

**Comment by RussKie** [comment]
The last two commits need to be squashed

**Comment by cryham** [comment]
Right, done.

**Pull request merged**